asciidoc/courses/aura-fundamentals/modules/3-services-tools/lessons/2-query/lesson.adoc

```diff
@@ -27,18 +27,23 @@ image::images/02_query_tool_start.jpg[querytool_start,width=500,align=center]
 
 The Query UI consists of the sidebar, the Cypher editor, and the result frame(s).
 
-As a good practice, you should remove the result-frames you don't want/need.
-
-Click on the three dots next to the frame title and select **Remove frame**.
 
    
+== Create query blocks using AI
+
+You can also create query blocks using AI by clicking the "Generate with AI" button. This will prompt you to enter a description of the data you want to query, and the AI will generate a Cypher query based on your description:
 
+video::https://cdn.graphacademy.neo4j.com/courses/aura-fundamentals/generate-query-ai.mp4[role="cdn", Generate Query AI, 350, 300]
 
 == Querying
 Start by exploring the database details.
 
 Click on the labels and relationship types under **Database Information** to see the number of nodes and relationships in the database.
 
+
+You can use the GenAI Query builder to create a query that returns all nodes and relationships in the database:
+video::https://cdn.graphacademy.neo4j.com/courses/aura-fundamentals/genai-query-builder.mp4[role="cdn",  GenAI Query Builder, 350, 300]
```

Comment:
This is the same video as the one above.

https://cdn.graphacademy.neo4j.com/courses/aura-fundamentals/generate-query-ai.mp4

They have different names but the content is the same.

Is the section on Generating AI queries duplicate?